docs/api/browser-window.md

```diff
@@ -835,7 +849,7 @@ Moves window to `x` and `y`.
 
 #### `win.getPosition()`
 
-Returns an array that contains window's current position.
+Returns `Integer[]` - Contains the window's current position.
```

Comment:
I've mentioned this below already, but I don't think we should be introducing any new syntax, just write "array of TYPE" or "TYPE array"... I think the parser can be modified to deal with both variants right @zeke (or just the first one if it makes things easier)?